### PR TITLE
feature: notifications when creating clients, matter and documents

### PR DESCRIPTION
**Pull request by yneves**

### CODE REVIEW

**Comment by yneves** [comment]
Added the toaster notifications to the creation of clients, matters and documents. Once we have it merged, we can extend it to other actions.

File: src/components/toasters/Toaster.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    let style = {
+      position: 'absolute',
+      height: this.props.height,
+      bottom: this.props.index * this.props.height,
+      width: '100%',
+      textAlign: 'right'
+    };
+    return (
+      <div className='toaster' style={style}>
+        <Fade in={!this.props.removed} onExited={::this.onExited}>
```
Comment (ntgn81) [inline src/components/toasters/Toaster.js]: Missing `removed` in propTypes

**Comment by ntgn81** [comment]
Looks good except for that 1 lint problem!

File: src/components/toasters/Toaster.js (hunk 1 above)
Comment (yneves) [inline src/components/toasters/Toaster.js]: Fixed

**Pull request merged**